src/styles/blocks/_lection.scss

```diff
@@ -0,0 +1,46 @@
+.lection {
+  &__photo {
+    background: url(../images/events/events-img-3.png) no-repeat;
+    background-size: cover;
+    background-position: center;
+    height: 210px;
+    width: 100%;
+    z-index: -1;
+
+    @include on-tablet {
+      height: 390px;
+      width: 700px;
+    }
+
+    @include on-desktop {
+      height: 550px;
+      width: 100vw;
```

Comment:
the problem is here, just change it to % and everything will be fixed

Reply:
ok, i will do it )))
But I seriously don't see scroll on my laptop now/